proposals/lip-0045.md

```diff
@@ -633,7 +665,8 @@ chainIDSchema = {
     "required": ["ID"],
     "properties": {
         "ID": {
```

Comment:
I would call this property `chainID`. As we handle different IDs here (e.g., also the network ID), this makes things more explicit.

Reply:
Again the description below is inconsistent with the schema:

> The store value is an object with chainID=MAINCHAIN_ID, serialized using the JSON schema chainIDSchema.

Reply:
I resolve this point as it is fixed.